src/controller/jobRoleController.ts

```diff
@@ -52,4 +52,36 @@ export namespace JobRoles {
             });
         }
     }
+
+    export async function getCreate(req:Request, res:Response): Promise<void> {
+        const bands: JobBand[] = await bandService.getBands();
+        const capabilities: JobCapability[] = await capabilityService.getCapabilities();
```

Comment:
What is the user experince if any of the services throws an exception?

Reply:
<img width="1232" alt="Screenshot 2023-09-26 at 13 37 26" src="https://github.com/anthony-raybould/bham1-front-end/assets/142424155/ddae5ec1-19b9-4ccf-8f34-8bfd23f7a3ee">